code/__HELPERS/unsorted.dm

```diff
@@ -1196,6 +1196,12 @@ var/global/list/common_tools = list(
 			return 0
 	if(istype(W, /obj/item/device/assembly/igniter))
 		return 1000
+	if(istype(W, /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass))

```

Comment:
This is probably going to create problems. I don't recommend doing this.

This proc is primarily used in `attackby` and some other things; overriding reagent container default behavior is generrrrallyy not a good thing.

As a prime example, trying to dump one reagent container into another, with this, will probably heat up the reagent container you attempt to dump it into.